docs/core/tools/dotnet-list-package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: dotnet list package command
+description: The 'dotnet list package' command provides a convenient option to list the package references for a project or solution.
+ms.date: 01/30/2019
+---
+# dotnet list package
+
+[!INCLUDE [topic-appliesto-net-core-22plus](../../../includes/topic-appliesto-net-core-22plus.md)]
+
+## Name
+
+`dotnet list package` - Lists the package references for a project or solution.
+
+## Synopsis
+
+```
+dotnet list [<PROJECT | SOLUTION>] package [--config] [--framework] [--highest-minor] [--highest-patch] [--include-prerelease] [--include-transitive] [--outdated] [--source]
+dotnet list package [-h|--help]
+```
+
+## Description
+
+The `dotnet list package` command provides a convenient option to list all NuGet package references for a specific project or a solution. You first need to build the project in order to have the assets needed for this command to run. The following example shows the output of the `dotnet list package` command for the [SentimentAnalysis](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/SentimentAnalysis) project:
+
+```output
+Project 'SentimentAnalysis' has the following package references
+   [netcoreapp2.1]:
+   Top-level Package               Requested   Resolved
+   > Microsoft.ML                  0.9.0       0.9.0
+   > Microsoft.NETCore.App   (A)   [2.1.0, )   2.1.0
+
+(A) : Auto-referenced package.
+```
+
+The **Requested** column refers to the package version specified in the project file and can be a range. The **Resolved** column lists the version that the project is currently using and is always a single value. The packages displaying an `(A)` right next to their names represent [implicit package references](csproj.md#implicit-package-references) that are inferred from your project settings (`Sdk` type, `<TargetFramework>` or `<TargetFrameworks>` property, etc.)
+
+Use the `--outdated` option to find out if there are newer versions available of the packages you're using on your projects. By default, `--outdated` lists the latest stable packages unless the resolved version is also a prerelease version. To include prerelease versions when listing newer versions, also specify the `--include-prerelease` option.
```

Comment:
Would it be worthwhile to add a sample output when using the --outdated option?

Reply:
yes, I think that would be interesting. If you're looking for a simple example, I think xunit is good. It's commonly used, has transitive dependencies, but not too many so the sample won't be too long.